apps/web-giddh/src/app/manufacturing/report/mf.report.component.scss

```diff
@@ -107,6 +120,10 @@
             flex: 0 0 66.666667%;
             max-width: 66.666667%;
         }
+        .col-xl-9 {
```

Comment:
@shalinee-walkover can you please change this column class by other class name, otherwise for every col-xl-9 it will work..